src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,443 @@
+</CodeExample>
+
+##### Cursor pagination
+
+Use [Cursor pagination](/components/pagination#cursor-pagination) for real-time data.
+
+- This includes when new records are added frequently or in data sets where the latest results are first
+- There is no concept of the total number of pages in the set
+- Use “Previous” and “Next” as pagination labels
+
+![Pagination Cursor](figma:patterns-tables-basic-formatting-pagination-cursor.png)
+
+##### Offset pagination
+
+Use [Offset pagination](/components/pagination#offset-pagination) when you know the number of records or where you are
+in the data set.
+
+- Ensure the margin between the table component and the pagination is 20px
+
+![Pagination Offset](figma:patterns-tables-basic-formatting-pagination-offset.png)
+
+##### Select all for pagination
+
+In paginated tables, you can select all checkboxes but that selection only affects the current page being shown.
+
+- Users should only be able to take action on what they can see
+
+![Pagination Selecting All](figma:patterns-tables-basic-formatting-pagination-selecting-all.png)
+
+#### Infinite scrolling
+
+In cases where the database supports infinite scrolling, there is no need to say how many pages of data there are.
+
+If you have an infinitely scrolling table, you should also offer [sorting](/patterns/tables/basic-formatting#sorting) to
+help users navigate the data.
+
+##### Select all for infinite scrolling
+
+In infinitely scrolling tables, you can only manually select items.
+
+##### Vertical scrolling
+
+For tables with more rows than can be displayed in the viewport, the header row is fixed at the top of the viewport.
+Keeping the column headers within view while scrolling helps users maintain context.
+
+import TableVerticalScrolling from '../examples/tables/TableVerticalScrolling.tsx';
+import TableVerticalScrollingCode from '!!raw-loader!../examples/tables/TableVerticalScrolling.tsx';
+
+<CodeExample code={TableVerticalScrollingCode}>
+  <TableVerticalScrolling />
+</CodeExample>
+
+##### Virtualization
+
+Virtualization means what you see in the browser is what is in the DOM. When something scrolls out of view, it gets
+removed from the page DOM. When something scrolls into view, it is added to the DOM.
+
+- Use [react-window](https://github.com/bvaughn/react-window) to efficiently render large data sets
+
+import TableVirtualized from '../examples/tables/TableVirtualized.tsx';
+import TableVirtualizedCode from '!!raw-loader!../examples/tables/TableVirtualized.tsx';
+
+<CodeExample code={TableVirtualizedCode}>
+  <TableVirtualized />
+</CodeExample>
+
+#### Sorting
+
+Sorting is implemented by selecting arrows in table headers.
+
+- Clicking the header text will sort by ascending order (A→Z, 1→9, Date and Time, Priority (High→Low))
+- Clicking again will reverse the order
+- Clicking a third time will remove sorting
+
+If any ordering applied to a table including default ordering, the sorting icon should be present. This indicates how
```

Comment:
This reads funny. I think we left out "is"? "If any ordering is applied to a table including default ordering, the sorting icon should be present." I'll update the docs too cause they were missing this.

Reply:
@diana-oum Thank you. Also updated the Ready to Build latest Figma to reflect this.